internal/config/profile.go

```diff
@@ -242,9 +245,9 @@ func (p *profile) IsAccessSet() bool {
 	return isSet
 }
 
-// GetConfigDescription returns a map describing the configuration
-func GetConfigDescription() map[string]string { return p.GetConfigDescription() }
-func (p *profile) GetConfigDescription() map[string]string {
+// Get returns a map describing the configuration.
+func Get() map[string]string { return p.Get() }
```

Comment:
description meant more of a text return but since we are returning a map, also GetConfig is somewhat of a stutter given this the config package